### PR TITLE
doc: initial data layer documentation

### PR DESCRIPTION
**Pull request by kldavis4**
Documentation for the experimental data layer functionality in Tina Cloud

### CODE REVIEW

File: content/docs/tina-cloud/data-layer.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Data Layer
+last_edited: '2022-02-15T20:00:00.000Z'
+---
+
+{{ WarningCallout text="This is an experimental feature, and the API is subject to change. Have any thoughts? Let us know in the chat, or through one of our [community channels](/community/)!" }}
+
+Tina Cloud now supports an optional Data Layer which implements a database layer between the Tina's GraphQL api and the
```
Comment (jhuggett) [inline content/docs/tina-cloud/data-layer.md:8]:
```suggestion
Tina Cloud now supports an optional Data Layer which implements a database layer between the Tina GraphQL api and the
```

File: content/docs/tina-cloud/data-layer.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+performance with TinaCMS. As this feature evolves, it will provide additional capabilities such as:
+- more sophisticated GraphQL queries
+- referential integrity
+- sorting
+- filtering
+- pagination 
+
+# Enabling the Data Layer
+
+The TinaCMS data layer is enabled by specifying the following TinaCMS command line flag: `--experimentalData`. We
```
Comment (jhuggett) [inline content/docs/tina-cloud/data-layer.md:19]: How do you feel about putting the flag in a code block? The way it's rendering now puts one dash on a separate line and I could imagine a user (ie me) trying to copy paste it and missing the first dash.

Comment (jhuggett) [reply content/docs/tina-cloud/data-layer.md:19]: actually never mind, it's there in the "start-tina" script. It'd be nice if the flag didn't break across lines like that but oh well

File: content/docs/tina-cloud/data-layer.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+recommend editing the script(s) in your site's package.json to pass the flag:
+
+```json
+"start-tina": "yarn tinacms server:start -c \"next start\" --experimentalData"
+```
+
+Once the flag is added and the TinaCMS cli is executed using the flag, the generated schema will be updated. These
+updates must be committed and pushed to the remote GitHub repository in order to activate the data layer.
```
Comment (jhuggett) [inline content/docs/tina-cloud/data-layer.md:27]:
```suggestion
Once the flag is added and the TinaCMS cli is executed using the flag, the generated schema will be updated. **These
updates must be committed and pushed to the remote GitHub repository in order to activate the data layer.**
```

**Review by jamespohalloran — DISMISSED** [review]
This looks good to me!

File: content/docs/tina-cloud/data-layer.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+---
+title: Data Layer
+last_edited: '2022-02-15T20:00:00.000Z'
+---
+
+{{ WarningCallout text="This is an experimental feature, and the API is subject to change. Have any thoughts? Let us know in the chat, or through one of our [community channels](/community/)!" }}
+
+Tina Cloud now supports an optional Data Layer which implements a database layer between the Tina GraphQL api and the
+GitHub rest api. This data layer buffers requests between Tina Cloud and GitHub resulting in improved editing
```
Comment (mittonface) [inline content/docs/tina-cloud/data-layer.md:9]:
```suggestion
Tina Cloud now supports an optional Data Layer which implements a database layer between the Tina GraphQL API and the
GitHub REST API. This data layer buffers requests between Tina Cloud and GitHub resulting in improved editing
```

File: content/docs/tina-cloud/data-layer.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+# Enabling the Data Layer
+
+The TinaCMS data layer is enabled by specifying the following TinaCMS command line flag: `--experimentalData`. We
+recommend editing the script(s) in your site's package.json to pass the flag:
+
+```json
+"start-tina": "yarn tinacms server:start -c \"next start\" --experimentalData"
+```
+
+Once the flag is added and the TinaCMS cli is executed using the flag, the generated schema will be updated. **These
```
Comment (mittonface) [inline content/docs/tina-cloud/data-layer.md:26]:
```suggestion
Once the flag is added and the TinaCMS CLI is executed using the flag, the generated schema will be updated. **These
```

**Review by mittonface — COMMENTED** [review]
Just capitalized from acronyms!

**Review by jamespohalloran — APPROVED** [review]

**Review by logan-anderson — APPROVED** [review]

**Review by jhuggett — APPROVED** [review]

**Pull request merged**